### PR TITLE
tests: add github actions

### PR DESCRIPTION
**Pull request by Bemilie**
## Summary

**_What's changed?_**

- Added github actions to run integration tests
- Updated tests to run in parallel

**_Why do we need this?_**

- To ensure that new features is not breaking any current feature

**_How have you tested it?_**

- GHA are triggered and all tests are passing

## Checklist

- [X] I have reviewed this myself
- [ ] There is a unit test covering every change in this PR
- [ ] I have updated the relevant documentation

## Details

### CODE REVIEW

File: tests/containers/containers_private_registry.test.js [hunk 1]
```diff
@@ -2,31 +2,37 @@
...
 const crypto = require('crypto');
 const Docker = require('dockerode');
-
-const docker = new Docker();
-
-const path = require('path');
 const fs = require('fs');
+const path = require('path');
+
 const { expect } = require('chai');
```
Comment (cyclimse) [inline tests/containers/containers_private_registry.test.js:8]: Are we using chai + jest? We could probably just use the assertions from jest

File: tests/multi-region/multi_region.test.js [hunk 2]
```diff
@@ -1,71 +1,82 @@
...
+  async (region) => {
+
+      let options = {};
+      options.env = {};
+      options.env.SCW_SECRET_KEY = scwToken;
+
+      let projectId;
+      let apiUrl;
+      let api;
+      let namespace;
```
Comment (cyclimse) [inline tests/multi-region/multi_region.test.js:39]: you can also do `let projectId, apiUrl, ...` but it doesn't really matter

However, I wonder if this codebase is linted? Now that we maintain more Javascript projects, we can try to have the same linter config between them.

File: .github/workflows/test.yml [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+
+      - name: Set up node ${{ matrix.node-version }}
+        id: setup-node
+        uses: actions/setup-node@v3
+        with:
+          node-version: ${{ matrix.node-version }}
+          cache: "npm"
+
+      - name: Install serverless framework
+        run: npm install -g serverless
```
Comment (cyclimse) [inline .github/workflows/test.yml:36]: Should we pin it in our dev dependencies?

File: package.json [hunk 4]
```diff
@@ -36,32 +39,33 @@
...
   "devDependencies": {
-    "@jest/globals": "^29.3.1",
+    "@jest/globals": "^29.5.0",
     "chai": "^4.3.7",
     "eslint": "^8.27.0",
     "eslint-config-airbnb-base": "^15.0.0",
     "eslint-plugin-import": "^2.26.0",
     "fs-extra": "^10.1.0",
-    "jest": "^29.3.1",
+    "jest": "^29.5.0",
```
Comment (cyclimse) [inline package.json:68]: Are we missing dependabot on this repo?

**Review by cyclimse — APPROVED** [review]
Awesome, really happy you got those tests working in the end. Great work 🥳

Just a few requests:
- dependabot, saw quite a few dependency update commits
- linter config and add linting and formatting in the new CI

**Review by thomas-tacquet — APPROVED** [review]

**Pull request merged**